business_requirement/models/business.py

```diff
@@ -110,7 +110,9 @@ class BusinessRequirement(models.Model):
     )
     priority = fields.Selection(
         [('0', 'Low'), ('1', 'Normal'), ('2', 'High')],
-        'Priority', select=True
+        'Priority',
+        required=True,
```

Comment:
@elicoidal If there is records on my database before the creation of priority field you will need an update (some SQL script) for this records (otherwise will be empty string on database).

To correct this, if I only put the Parameter default, the next records will have this value but previous still not.

If I add Parameter required also I update the previous records created.

To be sure I test it step by step.

cc @elicoidal @duanyp1991